.github/workflows/contracts.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 
       - name: Install Solidity
         env:
-          SOLC_VERSION: 0.7.6 # according to solidity.version in hardhat.config.js
+          SOLC_VERSION: 0.7.6 # according to solidity.version in hardhat.config.ts
```

Comment:
This may not necessary be in scope of this PR, but I noticed that the version here doesn't match what's in the `hardhat.config.ts` (`0.8.4` is set in there). Should we update the version here or maybe get rid of the comment?

Reply:
Good catch! We should update it to match the version from the file.

Reply:
Done as part of https://github.com/keep-network/coverage-pools/pull/155.